### ISSUE TITLE
AdditiveAbelianGroup_class.exponent() returns a Python int for the trivial group

### ISSUE THREAD

**Issue by yyyyx4** [sagemath/sage#34872]
<div id="comment:0"></div>

```sage
sage: type(AdditiveAbelianGroup([]).exponent())
<class 'int'>
```

Component: **group theory**

Author: **Lorenz Panny**

Branch/Commit: **[public/34872](https://github.com/sagemath/sagetrac-mirror/tree/public/34872) @ [`6d6e020`](https://github.com/sagemath/sagetrac-mirror/commit/6d6e02089ccf56760cb6f63f27f7ad4ccbb2d405)**

Reviewer: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34872_

**Comment by yyyyx4** [comment]
Branch: **[public/34872](https://github.com/sagemath/sagetrac-mirror/tree/public/34872)**

**Comment by yyyyx4** [comment]
Commit: **[`6d6e020`](https://github.com/sagemath/sagetrac-mirror/commit/6d6e02089ccf56760cb6f63f27f7ad4ccbb2d405)**

**Comment by yyyyx4** [comment]
Author: **Lorenz Panny**

**Comment by mkoeppe** [comment]
Reviewer: **Matthias Koeppe**

**Comment by mkoeppe** [comment]
Merged in 10.0.beta0
